setup.py

```diff
@@ -26,7 +26,7 @@ def package_data(pkg, roots):
 
 setup(
     name='xblock-sql-grader',
-    version='0.0.1',
+    version='0.0.2',
```

Comment:
No changes to the package itself were needed, and we're not deploying to PyPI yet, so probably don't need to increment the version.

Reply:
Reverted